### ISSUE TITLE
Live streaming not working on Samsung A53 (Android 12)

### ISSUE THREAD

**Issue by gianlu33** [dji-sdk/Mobile-SDK-Android#1187]
Hi,

The live stream feature on the Sample App is not working on my Samsung A53 with Android 12. I keep getting error -3 and the video is not visible from the LiveStreamView.

I used three different DJI drones (Mavic Mini, Mini 2, and Mavic Air) but I keep getting the same problem. I also tried with other Android smartphones (a Xiaomi Redmi Note 5 with Android 9, and a OnePlus Nord with Android 11) and everything works fine.

I still don't know if it's a problem of my Samsung smartphone or Android 12, any hints would be appreciated.

Thanks!

**Comment by DJI-William** [comment]
-3 means cannot get the video stream from the surface. Fro Samsung A53, are you able to see the video stream on the live streaming page.

**Comment by gianlu33** [comment]
No, using my Samsung A53 I am not able to see the video stream on the live streaming page.

I know what the error -3 means, the question is: how to fix this problem? I am using the Sample App so I assume the code is correct.

**Comment by DJI-William** [comment]
We have to find a Samsung A53 to test with it. Please use other Android device for now.

**Comment by gianlu33** [comment]
Thank you. So this is a problem of this specific smartphone? Do you know if the streaming is working on other Android 12 smartphones?

**Comment by DJI-William** [comment]
Therotically the live stream feature shouldn't not be smartphone model dependent and we have never encounter such an issue before. So in our knowledge the live stream should work with every Android devices in any API version.

**Comment by mitchrsm** [comment]
any progress on this? I encounter the same problem, with A53, also with a Samsung S22 Ultra

**Comment by DJI-William** [comment]
I think the problem is there is no video stream for these devices. If there is no video stream for these devices, you cannot live stream them as well. We are working on those smart phones now. They are using special chipsets.
